src/module/item/feat/sheet.ts

```diff
@@ -44,10 +53,20 @@ class FeatSheetPF2e extends ItemSheetPF2e<FeatPF2e> {
             });
         }
 
+        htmlQuery(html, "a[data-action=delete-effect")?.addEventListener("click", () => {
+            if (this.item._source.system.selfEffect) {
+                this.item.update({ "system.-=selfEffect": null });
+            }
+        });
```

Comment:
Stuff like these should probably be in the ability helpers, since they're going to be fully shared I imagine.

Reply:
You are a crazy person